Source/NSURL.m

```diff
@@ -587,6 +587,21 @@ + (id) fileURLWithPath: (NSString*)aPath isDirectory: (BOOL)isDir
 					    isDirectory: isDir]);
 }
 
++ (id) fileURLWithPath: (NSString *)aPath
+	      isDirectory: (BOOL)isDir
+	    relativeToURL: (NSURL *)baseURL
+{
+  return AUTORELEASE([[NSURL alloc] initFileURLWithPath:aPath
```

Comment:
I really prefer to have a space after the colon. Not sure whether the GNUstep ObjC guidelines required this, but it would be in line with the rest of the file.